openlibrary/api.py

```diff
@@ -23,9 +23,8 @@
 import web
 import logging
 import requests
-
 import six
-from six.moves.configparser import ConfigParser
+import configparser
```

Comment:
```suggestion
from configparser import ConfigParser
```
and then please revert line 80.

Reply:
Oh yeah :facepalm: